frontend/src/main/java/com/google/tripmeout/frontend/PlaceVisitStorage.java

```diff
@@ -0,0 +1,10 @@
+package com.google.tripmeout.frontend;
+
+import com.google.tripmeout.frontend.PlaceVisitModel;
+
+public interface PlaceVisitStorage {
+  public void addPlaceVisit(PlaceVisitModel placeVisit);
```

Comment:
Can you add some comments about what the expected behaviors are for these methods?
- For `addPlaceVisit` what is the expected behavior of implementations when the place already exists? Does it replace the existing object? Does it throw an exception? Does it fail silently?
- For `removePlaceVisit`, `getPlaceVisit` and `changePlaceVisitStatus` what is the expected behavior of implementations when the place doesn't exist?